types/packages/caver-transaction/src/transactionTypes/legacyTransaction/legacyTransaction.d.ts

```diff
@@ -0,0 +1,38 @@
+/*
+    Copyright 2021 The caver-js Authors
+    This file is part of the caver-js library.
+    The caver-js library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    The caver-js library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import AbstractTransaction, { CreateTransactionObject } from '../abstractTransaction'
+
+export default class LegacyTransaction extends AbstractTransaction {
+    static create(createTxObj: CreateTransactionObject): LegacyTransaction
+    static decode(rlpEncoded: string): LegacyTransaction
+
+    constructor(createTxObj: CreateTransactionObject)
+
+    appendSignatures(sig: string[] | string[][]): void
+    getRLPEncoding(): string
+    getRLPEncodingForSignature(): string
+    getCommonRLPEncodingForSignature(): string
+    recoverPublicKeys(): string[]
+
+    private _to: string
+    public to: string
+    private _value: string
+    public value: string
+    private _input: string
+    public input: string
+    private _data: string
```

Comment:
I thought this field didn't define in LeagcyTransaction.

Reply:
In deprecated LegacyTrasnaction type and also Contract uses `data` filed, so in data setter, set value to the `input` in this case.